controllers/grid/issues/form/IssueForm.inc.php

```diff
@@ -65,7 +65,8 @@ function fetch($request) {
 		}
 
 		// Cover image preview
-		$coverImage = $this->issue->getCoverImage();
+		$coverImage = null;
+		if ($this->issue) $coverImage = $this->issue->getCoverImage();
```

Comment:
A neat little one-liner for this is:

```
$coverImage = $this->issue ? $this->issue->getCoverImage() : null;
```

Neither way's better. Just sharing. :)